src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,20 @@
+.watch {
+  height: $watch-size;
+  width: $watch-size;
+  border: 5px dotted black;
+  border-radius: 50%;
+  align-self: center;
+  display: flex;
+  justify-content: center;
+
+  &__center {
+    align-self: center;
+    position: absolute;
+  }
+
+  &__minute-axis {
```

Comment:
think this whole class should be moved to `minute-axis.scss` and following to BEM convention used as `.minute-axis--center` because it's just a modifier for centering an element